src/sentry/apidocs/build.py

```diff
@@ -104,6 +104,16 @@ def get_old_json_components(filename: str) -> Any:
             "url": "https://github.com/getsentry/sentry-docs/issues/new/?title=API%20Documentation%20Error:%20/api/integration-platform/&template=api_error_template.md",
         },
     },
+    {
+        "name": "Dashboards",
+        "x-sidebar-name": "Dashboards",
+        "description": "Endpoints for Dashboards",
+        "x-display-description": True,
```

Comment:
Hmm, what does this attribute do? I noticed a majority of the tags have this  set to false instead.

Reply:
![Screenshot 2024-07-17 at 11 36 25 AM](https://github.com/user-attachments/assets/1592113d-1e49-4de6-ae25-da857a784a11)
it displays this description text. I'm going to remove it here.